### ISSUE TITLE
[Bug] Unlock Firefox Lockwise pop-up not accessible via swipe using Talkback

### ISSUE THREAD

**Issue by asadotzler** [mozilla-mobile/fenix#9905]
## Steps to reproduce
Open a web page with a login field
Visually a "Unlock Firefox Lockwise" pop-up appears attached to the login field
Try swiping through the page to reach the Lockwise pop-up
Try explore by touch to locate the Lockwise pop-up

### Expected behavior
The Lockwise pop-up receives focus and "Unlock Firefox Lockwise" is read aloud. 

### Actual behavior
The Lockwise pop-up is invisible to swipes and when you find it using explore by touch, it is read as "search percent one singapore dollar" instead of "Unlock Firefox Lockwise". 

Also note that when you've unlocked Lockwise, the autocomplete pop-up receives focus but if you lose focus on the autocomplete pop-up you cannot get it back with swipes and must locate it with explore by touch

### Device information

* Android device: Galaxy S10e
* Fenix version: 4/8 (2020) build

This is an accessibility P1 because the Feature is almost completely broken for the screen reader user.

**Comment by mcarare** [comment]
I do not think that pop-up is something we do from Fenix, but rather from Lockwise. 
Also, the string with the content description you mentioned is from Lockwise.

**Comment by ekager** [comment]
@mcarare submitted a fix to Lockwise for this bug that has been merged so whenever (?) there's a new Lockwise version, this should be fixed.

**Comment by asadotzler** [comment]
Great! I'll verify and close this issue as soon as I get an updated Lockwise.

**Comment by mcarare** [comment]
I see a new version in Playstore, from 29 June 2020. Can QA please check if this issue is fixed? TY!

**Comment by softvision-miralobontiu** [comment]
Verified as fixed on the latest Nightly build from 7/20, and 4.0.1 (6349) Lockwise build.
Tested with OnePlus 5T (Android 9), and Google Pixel (Android 10).
